# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: servo/ports/geckolib/glue.rs
# Commit: acdd013202e1
# Full Hash: acdd013202e1e688986e7e6c747b7c7fee38474e
# Author: Boris Chiou <boris.chiou@gmail.com>
# Date: 2022-08-23 02:46:57
# Regressor Bug: 1694741
# File Overlap Count: 1
# Description:
#   Bug 1694741 - Part 6: Map width/height attributes to the style of img elements. r=emilio
#   
#   This patch will use the width/height attributes from <source> to override
#   width/height/aspect-ratio CSS property values of <img> elements.
#   
# ==============================================================================

diff -r 2b4d9516b882 -r acdd013202e1 servo/ports/geckolib/glue.rs
--- a/servo/ports/geckolib/glue.rs	Mon Aug 22 20:18:39 2022 +0000
+++ b/servo/ports/geckolib/glue.rs	Mon Aug 22 20:18:39 2022 +0000
@@ -5471,6 +5471,7 @@
         MarginRight => auto,
         MarginBottom => auto,
         MarginLeft => auto,
+        AspectRatio => specified::AspectRatio::auto(),
     };
     write_locked_arc(declarations, |decls: &mut PropertyDeclarationBlock| {
         decls.push(prop, Importance::Normal);